src/routers/messages.py

```
@@ -25,286 +35,113 @@
+    """Parse form data for file upload requests"""
+    return schemas.MessageUploadCreate(peer_id=peer_id)
 
-            logger.debug(f"Inserting {len(queue_records)} queue records")
 
-            if len(queue_records) > 0:
-                # Use insert to maintain order
-                stmt = insert(QueueItem).returning(QueueItem)
-                await db_session.execute(stmt, queue_records)
-                await db_session.commit()
+@router.post("/", response_model=list[schemas.Message])
+async def create_messages_for_session(
```

Ruff (0.12.2)

Missing return type annotation for public function `create_messages_for_session`

(ANN201)